mathics/builtin/drawing/discrete_plot.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+"""
+Discrete Plot
+"""
+import numbers
+
+from functools import lru_cache
+from typing import Optional
+
+from mathics.builtin.drawing.plot import (
+    Plot,
+    check_plot_range,
+    compile_quiet_function,
+    get_plot_range,
+)
+from mathics.builtin.graphics import Graphics
+from mathics.builtin.options import options_to_rules
+
+from mathics.core.attributes import A_HOLD_ALL, A_PROTECTED
+from mathics.core.convert.python import from_python
+from mathics.core.expression import Expression
+from mathics.core.list import ListExpression
+from mathics.core.symbols import Symbol, SymbolList
+from mathics.eval.nevaluator import eval_N
+from mathics.eval.plot import eval_ListPlot
+
+
+class _DiscretePlot(Plot):
+
+    attributes = A_HOLD_ALL | A_PROTECTED
+
+    expect_list = False
+
+    messages = {
+        "prng": (
+            "Value of option PlotRange -> `1` is not All, Automatic or "
+            "an appropriate list of range specifications."
```

Comment:
This is a bit confusing, but I don't know how to reformulate it

Reply:
What aspect is confusing. Describing the problem well sometimes is harder than solving it once the problem is pinpointed.

Reply:
What is "range specifications"? "Specifications" isn't used as much as "options" in other places, so maybe "range options"?

Reply:
I just had a chance to look at this. The wording of this message comes from WMA (it allows more options than we do, but it ends in "or an appropriate list of range specifications".   

Given that this is the way WMA words this, we shouldn't try to change the wording.